sdk/src/slot_history.rs

```diff
@@ -54,19 +95,111 @@ impl SlotHistory {
 #[cfg(test)]
 mod tests {
     use super::*;
+    use log::*;
 
     #[test]
-    fn test() {
+    fn slot_history_test1() {
+        solana_logger::setup();
+        // should be divisable by 64 since the clear logic works on blocks
+        assert_eq!(MAX_ENTRIES % 64, 0);
         let mut slot_history = SlotHistory::default();
+        info!("add 2");
         slot_history.add(2);
         assert_eq!(slot_history.check(0), Check::Found);
         assert_eq!(slot_history.check(1), Check::NotFound);
         for i in 3..MAX_ENTRIES {
             assert_eq!(slot_history.check(i), Check::Future);
         }
+        info!("add 20");
+        slot_history.add(20);
+        info!("add max_entries");
         slot_history.add(MAX_ENTRIES);
```

Comment:
;)

Reply:
Actually this test was not that slow. It was the `test_credit_debit_rent_no_side_effect_on_hash` which adds a slot that is 157 million slots in the future.